domain/src/main/java/edu/nextstep/camp/calculator/domain/Expression.kt

```diff
@@ -4,21 +4,22 @@ import edu.nextstep.camp.calculator.domain.util.CalculatorInputUtil
 
 object Expression {
     var express: String = ""
+
```

Comment:
```suggestion
    var express: String = ""
        private set
```
외부에서 변경이 불가능하도록 하려면, 내부에서만 set이 되도록 하는 것이 좋아요.

Reply:
궁금한 점에서 작성해주신 `private(set) var express` 이 부분은 어떤 것을 의미하는지 잘 모르겠네요 😭 
프로퍼티에 대해서 학습해보시면 좋을 것 같아요

ref. https://kotlinlang.org/docs/properties.html#getters-and-setters

Reply:
감사합니다!